### PR TITLE
Splitting lesson into different pages

### PR DESCRIPTION
**Pull request by mjones01**
The two previous files (2015-12-03-Colorado-Flood-Pre-Classroom.Rmd  &  2015-12-03-Colorado-Flood-Introduction.Rmd) have been split into six different files to cover the individual parts of the lessons.  All lessons renamed Quantifying Disturbance Events...   
1) QDE: Disturbance Event Overview
2) QDE: 2013 Colorado Flood Overview
3) QDE: Return Interval  (could be folded back into one of previous lessons if desired)
4) QDE Drivers & Impacts: Precip & Discharge
5) QDE Drivers & Impacts: LiDAR
6) QDE Drivers & Impacts: Imagery

The text within the files has been cut & pasted to the appropriate files but no editing for continuity, overlaps, or gaps has occurred.  There is currently no information in the Imagery file.

### CODE REVIEW

**Comment by lwasser** [comment]
Thank you, Megan

One page is one i'm using today to present.

https://github.com/NEONInc/NEONDS-Template-Repository/blob/gh-pages/_posts/boulder-flood/2015-12-03-Colorado-Flood-Introduction.Rmd

Can you please copy the file into your file and resubmit the PR so i can keep that page as is?
I understand you might have transferred some content - that is fine if it's in two places for now!

And it's fine to do it later. I'm not accepting the PR until it's fixed - that way i can keep my page in tact as a record :) 

Thank you
leah

**Comment by mjones01** [comment]
@lwasser The page from today's lesson has a been added to the pull request.

File: _posts/boulder-flood/QuantifyDistEvents_Drivers&Impacts_2LiDAR.Rmd [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
```
Comment (lwasser) [inline _posts/boulder-flood/QuantifyDistEvents_Drivers&Impacts_2LiDAR.Rmd]: @mjones01 please adjust the file name. placing none letter characters in file names can often be problematic. Thank you. please use drivers_impacts

**Comment by mjones01** [comment]
@ lwasser.  Updated.

**Comment by lwasser** [comment]
Thank you! i will finish my agu poster and then merge. looks like there are a few conflicts for me to sort through :) Very excited to work on this module. I think it will be a cool set of lessons.

**Pull request merged**